cookbook/specs/spec_add_cosmossdk.json

```diff
@@ -4299,6 +4333,20 @@
                                     "encoding": "hex"
                                 },
                                 "api_name": "block"
+                            },
+                            {
+                                "function_template": "{\"jsonrpc\":\"2.0\",\"method\":\"status\",\"params\":[],\"id\":1}",
```

Comment:
we set up the whole thing here and it is never used except for the function_tag.
we need to find a more elegant place to mark that it supports earliest